<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auth</class>
 <widget class="QWidget" name="Auth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Продолжить</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LoginEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
